### PR TITLE
Fix/discord

### PR DESCRIPTION
**Pull request by Awarua-**
update for latest discord.js api

### CODE REVIEW

File: core/core_integrations/discord.js [hunk 1]
```diff
@@ -108,73 +59,79 @@ let discord = require('discord.js'),
...
         let channel = lookUpChannel(threadId);
-        message = addMentions(message);
-        message = sentenceSplitter(message);
-        sendMultipleMessages(message, threadId, channel);
+        channel.stopTyping();
+        switch (type) {
+        case 'file':
+        case 'url':
+            channel.sendFile(file, '', description);
+            break;
```
Comment (mrkno) [inline core/core_integrations/discord.js]: has this even been tested? you are trying to send both a string url and a bytearray file using the same function call. Also what is the point of a switch where there are only 2 actual cases? if-else?

File: core/core_integrations/discord.js [hunk 2]
```diff
@@ -108,73 +59,79 @@ let discord = require('discord.js'),
...
     },
 
-    sendFile = function(type, file, description, threadId) {
+    sendUrl = (url, threadId) => {
         let channel = lookUpChannel(threadId);
-        stopTyping(threadId);
-        bot.sendFile(channel, file, description);
+        channel.stopTyping();
+        channel.sendFile(url);
     },
```
Comment (mrkno) [inline core/core_integrations/discord.js]: sendFile and yet no sendImage?

File: core/core_integrations/discord.js (hunk 1 above)
Comment (Awarua-) [inline core/core_integrations/discord.js]: Yes this has been tested. According to Discord.js api sendFile supports Strings and Buffers. https://hydrabolt.github.io/discord.js/#!/docs/tag/master/typedef/FileResolvable

Changed logic to if-else;

File: core/core_integrations/discord.js (hunk 2 above)
Comment (Awarua-) [inline core/core_integrations/discord.js]: sendImage is mapped to sendFile, as they are internally the same logic

**Pull request merged**